src/components/sections/FeaturedPools.vue

```diff
@@ -5,7 +5,8 @@
       <div class="col-span-12 lg:col-span-6">
         <BalCard
           title="Element Finance"
-          imgSrc="/images/banners/element.png"
+          imgSrc="/images/banners/element.jpg"
+          class="transition-shadow hover:shadow-xl"
```

Comment:
The shadow class needed to be applied to the card rather than the grid box. Also, the card already had the `shadow-lg` class so I needed to use xl to see any affect. There was an effect before because applying it to the grid box was essentially applying the same shadow twice.